app/utils/url.js

```diff
@@ -98,8 +98,8 @@ export function getScheme(url) {
     return match && match[1];
 }
 
-export function matchDeepLink(url, serverURL, siteURL) {
-    if (!url || !serverURL || !siteURL) {
+export function matchDeepLink(url, serverURL = '', siteURL = '') {
```

Comment:
wouldn't the regex match now match something that shouldn't?
while I find fine having `''` as default value to prevent errors on escapeRegex (althought it might be its responsability to handle), we might want to remove the serverURL/siteURL if it is this default value

Reply:
nice catch, I was testing this and forgot to remove it.. just re-tested by removing the defaults and it works just fine